src/Faker/Container/ContainerBuilder.php

```diff
@@ -59,8 +59,19 @@ public static function defaultExtensions(): array
         ];
     }
 
+    public static function default(): self
+    {
+        $instance = new self();
+
+        foreach (self::defaultExtensions() as $id => $definition) {
+            $instance->add($id, $definition);
+        }
+
+        return $instance;
+    }
+
     public static function getDefault(): ContainerInterface
     {
-        return new Container(self::defaultExtensions());
+        return self::default()->build();
     }
```

Comment:
We can probably remove this method - I don't see any use for it (also see changes below).